Diff.java

```diff
@@ -0,0 +1,11 @@
+class Diff{
+    public static void main(String[] args) {
+        int x=100;
+        int y=20;
+
+        int diff;
```

Comment:
Please follow a variable naming convention.

Reply:
Hi 
Updation done to program

Reply:
ok let me check